src/fmgc/src/guidance/lnav/LnavDriver.ts

```diff
@@ -423,10 +424,12 @@ export class LnavDriver implements GuidanceComponent {
         // FIXME should be NCD if no FM position
 
         SimVar.SetSimVarValue('L:A32NX_EFIS_L_TO_WPT_BEARING', 'Degrees', efisBearing);
+        SimVar.SetSimVarValue('L:A32NX_EFIS_L_TO_WPT_TRUE_BEARING', 'Degrees', efisTrueBearing);
```

Comment:
Is there a reason we have to transmit both? Since it's only used for ND display.

Reply:
We either need to fo that, or transmit the magnetic variation to use.